stdlib/public/Concurrency/Task.swift

```diff
@@ -186,6 +186,42 @@ extension Task: Equatable {
   }
 }
 
+@available(SwiftStdlib 5.9, *)
+extension Task {
+    @_spi(MainActorUtilities)
+    @MainActor
+    public static func startOnMainActor(
+        priority: TaskPriority? = nil,
+        _ work: @escaping @MainActor() async throws -> Success
+    ) -> Task<Success, Error> {
+        let flags = taskCreateFlags(priority: priority, isChildTask: false,
+                                    copyTaskLocals: false, inheritContext: true,
```

Comment:
oh no, wait, the `copyTaskLocals` should be `true` here; we're semantically continuing work so we should not "effectively _detach_" here